src/middleware.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI, Request
+
+app = FastAPI()
```

Comment:
Did you test it? It doesn't work for me even using main app (as it should be). But since this file is not imported while creating app it doesn't see this middleware. So we have 2 ways how to fix it: 
1) create class-based middleware, import it to `main.py` and use `app.add_middleware()` function to include it in the middleware list. This is kind of advanced way. It would be good, but not required, we would be good with a simpler option;
2) Do you know how to apply decorator without `@` usage?) Btw, it is a question from one of my interviews) The second way - just remove the decorator, import this function to the `main.py` and after `include_router` apply middleware decorator to the middleware function